packages/charting/src/components/StackedBarChart/StackedBarChart.base.tsx

```diff
@@ -127,12 +153,39 @@ export class StackedBarChartBase extends React.Component<IStackedBarChartProps,
     });
   }
 
+  private _getTotalChartData(data: IChartProps): number {
+    let sum = 0;
+    if (data.chartData && data.chartData.length > 0) {
+      const chartData = data.chartData;
+      for (let i = 0, n = chartData.length; i < n; i++) {
+        sum += chartData[i].data || 0;
+      }
+    }
+
+    return sum;
+  }
+
+  private _createBenchmark(totalData: number, dataPoint: IChartDataPoint): JSX.Element {
+    const ratio = (dataPoint.data! / totalData) * 100;
+
+    const styles = {
+      marginLeft: 'calc(' + ratio + '% - 4.5px)',
+      marginRight: 'calc(' + (100 - ratio) + '% - 4.5px)',
```

Comment:
Does this handle RTL?

Reply:
change this code block to `left: 'calc(' + targetRatio + '% - 4.5px)'` in StackedBarChart.styles.ts.